golang/cosmos/x/vstorage/keeper/querier.go

```diff
@@ -0,0 +1,65 @@
+package keeper
+
+import (
+	"strings"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/Agoric/agoric-sdk/golang/cosmos/x/vstorage/types"
+)
+
+// query endpoints supported by the vstorage Querier
+const (
+	QueryData     = "data"
+	QueryChildren = "children"
+)
+
+// NewQuerier is the module level router for state queries
+func NewQuerier(keeper Keeper, legacyQuerierCdc *codec.LegacyAmino) sdk.Querier {
+	return func(ctx sdk.Context, path []string, req abci.RequestQuery) (res []byte, err error) {
+		switch path[0] {
+		case QueryData:
+			return queryData(ctx, strings.Join(path[1:], "/"), req, keeper, legacyQuerierCdc)
+		case QueryChildren:
+			return queryKeys(ctx, strings.Join(path[1:], "/"), req, keeper, legacyQuerierCdc)
```

Comment:
I'm liking "query children" as a consistent name to use throughout.

Reply:
Okay, I'll see about making that change.